src/Interactive/HostProcess/InteractiveHostEntryPoint.cs

```diff
@@ -15,6 +16,7 @@ namespace Microsoft.CodeAnalysis.Interactive
 {
     internal static class InteractiveHostEntryPoint
     {
+        [SupportedOSPlatform("windows")]
```

Comment:
Can this be surrounded by `#if NET5_0` instead of having to add the PlatformAttributes.cs file? or this won't work?

Reply:
I prefer to localize the directives and have the code itself look uniform and, where possible, modern.